### ISSUE TITLE
Accessibility issues: Analytical table

### ISSUE THREAD

**Issue by Bilal3120** [SAP/ui5-webcomponents-react#3121]
**Describe the bug**
The column header is not announced when reading out the data inside the table column.

**To Reproduce**
1. Go to [StoryBook](https://sap.github.io/ui5-webcomponents-react/?path=/docs/data-display-analyticaltable--default-story)
2. Select any Name

**Expected behavior**
The header along with the text is expected to be announced

**Screenshots**
![17512213-C855-4336-92FC-206A0D690FBE_1_201_a](https://user-images.githubusercontent.com/93638146/175472209-c2f24242-80c4-4761-a82f-f80f85983800.jpeg)

**UI5 Web Components for React Information**
`@ui5/webcomponents` 1.3.1
`@ui5/webcomponents-react` 0.23.2

Operating System: Mac Os  12.4

**Additional context**
Screen reader - Apple voice over
